.travis.yml

```diff
@@ -26,3 +33,5 @@ before_script:
   - make -j$(nproc)
   - export PATH=${PWD}:${PATH}
   - cd "${OLDPWD}"
+after_script:
+  - sleep 10
```

Comment:
How come?

Reply:
When I first tried two jobs, I wasn't getting saved full output until I added that. Looks to be a Travis issue, see the commit message for the link.

Reply:
Sadly...ok...does cutting this down to `sleep 1` work as reported: https://github.com/travis-ci/travis-ci/issues/6018#issuecomment-312066416 rather than 10?

Reply:
I'll try, the last commit reference before mine in that issue used 10 so that's what I went with.